json-rpc/API-CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ Please add the API change in the following format:
 
 ```
 
+## 2021-03-18 Add optional parameter `version` to `get_account` method
+
+Get account data by version.
+
+
```

Comment:
extra whitespace?

Reply:
one extra line won't have difference after rendered. sometime I just gave 2 lines at the end in MD for better reading.

Reply:
extra line in md file is fine when rendered. I added one extra line at the end of section for easier reading.